data.sql

```diff
@@ -13,3 +13,18 @@ OVERRIDING SYSTEM VALUE
   (9, 'Boarmon', '2005-06-07', 7, true, 20.4),
   (10, 'Blossom', '1998-10-13', 3, true, 17),
   (11, 'Ditto', '2022-05-14', 4, true, 22);
+
+INSERT INTO owners (full_name, age)
+VALUES ('Sam Smith', 34), ('Jennifer Orwell', 19), ('Bob', 45), ('Melody Pond', 77), ('Dean Winchester', 14), ('Jodie Whittaker', 38);
+
+INSERT INTO species (name)
+VALUES ('Pokemon'), ('Digimon');
+
+UPDATE animals SET species_id = 2 WHERE name LIKE '%mon';
+UPDATE animals SET species_id = 1 species_id IS NULL;
+
+UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
+UPDATE animals SET owner_id = 2 WHERE name IN ('Gabumon', 'Pikachu');
+UPDATE animals SET owner_id = 3 WHERE name IN ('Devimon', 'Plantmon');
+UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
+UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon', 'Boarmon');
```

Comment:
Hello @BenMukebo. Thanks very much for that suggestion. I just wanted to say that I really like your hard job. You really are an example of a good reviewer. 🙏🙏
I see your point it looks a little hard to understand at first but I see how helpful it will be if you have bigger table.

Reply:
Thanks @Muhammad0602, I appreciate 🙏🏻 🙏🏻